app/js/scripts.js

```diff
@@ -1522,46 +1616,48 @@ function changeReportView(deviceType, operation){
   updateQueryStringParam('reportType', deviceType);
   updateQueryStringParam('type', operation);
   //if it's view, then remove the add criteria buttons and save
-if (operation == 'edit') {
+  if (operation == 'edit') {
     $(".report-edit-button").show();
     $(".report-create-button").hide();
   } else {
+    $("#show-in-dashboard").prop('checked', false);
     $(".report-create-button").show();
     $(".report-edit-button").hide();
   }
 }
 //resets all url params that is not navigation based
-function resetURLParams(){
-  updateQueryStringParam('type',null);
-  updateQueryStringParam('serial',null);
-  updateQueryStringParam('udid',null);
-  updateQueryStringParam('reportType',null);
-  updateQueryStringParam('isUpdate',null);
-  updateQueryStringParam('type',null);
-  updateQueryStringParam('reportId',null);
+function resetURLParams() {
+  updateQueryStringParam('type', null);
+  updateQueryStringParam('serial', null);
+  updateQueryStringParam('udid', null);
+  updateQueryStringParam('reportType', null);
+  updateQueryStringParam('isUpdate', null);
+  updateQueryStringParam('type', null);
+  updateQueryStringParam('reportId', null);
 }
 
 //Wait for the page to render
-$(document).ready(function(){
-  if (!("auth_token" in sessionStorage)){
-    $("#login-button").click(function(){
+$(document).ready(function () {
+  if (!("auth_token" in sessionStorage)) {
+    $("#login-button").click(function () {
       doLoginLDAP();
     });
-    $("#login-user-button").click(function(){
+    $("#login-user-button").click(function () {
       doLoginUserPass();
     });
-    $("#register-button").click(function(){
+    $("#register-button").click(function () {
       registerUser();
     });
     $('#login-user-modal').modal('show');
     //Make sure reshow the login if they click out of it
     $('#login-user-modal').on('hidden.bs.modal', function () {
-      if (!("auth_token" in sessionStorage)){
+      if (!("auth_token" in sessionStorage)) {
         $('#login-user-modal').modal('show');
       }
     })
   } else {
     renderPage();
   }
 
+
```

Comment:
This is a pretty big diff - I assume this just went through an auto formatter? Was any regression testing performed to make sure this didn't break anything? Should probably be fine, just want to double check.

Reply:
It did. I'll revert it if you prefer.

Reply:
Nope, that's okay. Just wanted to double check. Probably makes it cleaner in the long run anyway.